galaxy/importer/models.py

```diff
@@ -186,13 +193,38 @@ def _is_valid_license_id(license_id, valid_license_ids):
         return True
 
     @dependencies.validator
-    def _check_dependencies_type(self, attribute, value):
-        '''Check type of dependencies dictionary and its contents'''
-        if not isinstance(value, dict) or value is None:
-            self.value_error("Expecting '%s' to be a dict" % attribute.name)
-        for k, v in value.items():
-            if not isinstance(k, str) or not isinstance(v, str):
-                self.value_error("Expecting dict key and value to be strings")
+    def _check_dependencies_format(self, attribute, dependencies):
+        '''Check type and format of dependencies collection and version'''
+        if not isinstance(dependencies, dict) or dependencies is None:
```

Comment:
Can't these type checks be expressed in attrs?

Reply:
Incorporated